### PR TITLE
New package: novaquark.myDU version 1.0

### PR DESCRIPTION
**Pull request by solomoncyj**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/171557)

### CODE REVIEW

**Comment by solomoncyj** [comment]
unhelpful errors

**Comment by stephengillie** [comment]
>  The file was not found. Verify git path must match with the casing of the file in disk. Not found file on 'manifests\n\novaquark\myDU\1.0\novaquark.myDU.installer.yaml'

There is [already](https://github.com/microsoft/winget-pkgs/tree/master/manifests/n/Novaquark/DualUniverse/1.0) a `Novaquark`, so this manifest's caps conflicts with the existing caps.

**Comment by solomoncyj** [comment]
image

**Comment by solomoncyj** [comment]
cant seem to find anything from the picture in the logs

**Comment by solomoncyj** [comment]
image

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 895.)

File: manifests/n/Novaquark/myDU/1.0/Novaquark.myDU.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Novaquark.myDU
+PackageVersion: "1.0"
+InstallerType: nullsoft
+Installers:
```
Comment (stephengillie) [inline manifests/n/Novaquark/myDU/1.0/Novaquark.myDU.installer.yaml:7]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Docker.DockerDesktop
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @solomoncyj,

It appears that `Docker.DockerDesktop` is a dependency:
image

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 895.)

**Comment by stephengillie** [comment]
Hi @solomoncyj,

Is there an installer switch to bypass the prompt and continue silently?

image

**Comment by solomoncyj** [comment]
it may be bacuse due to hyper-v not being active. docker requires hyper-v to run. try running `bcdedit /set hypervisorlaunchtype outo`

File: manifests/n/Novaquark/myDU/1.0/Novaquark.myDU.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Novaquark.myDU
+PackageVersion: "1.0"
+InstallerType: nullsoft
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Docker.DockerDesktop
```
Comment (stephengillie) [inline manifests/n/Novaquark/myDU/1.0/Novaquark.myDU.installer.yaml:9]:
```suggestion
    - PackageIdentifier: Docker.DockerDesktop
  WindowsFeatures:
    - Microsoft-Hyper-V
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
I'm hoping that this has the same effect.

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 8:38:25 AM: 
> 2024-09-12 08:38:24.815 [CORE] Failed to enable Windows Feature [Microsoft-Hyper-V] with exit code: 14081

Automated error analysis suggests 14081 may mean The referenced assembly could not be found. Is a dependency missing?

(Automated response - build 895.)

**Comment by Trenly** [comment]
@stephengillie - What is needed on this PR? Seems it's been neglected

**Comment by stephengillie** [comment]
This comment got buried by automated retries:

> Manual Validation ended with:
> 
> > 8:38:25 AM:
> > 2024-09-12 08:38:24.815 [CORE] Failed to enable Windows Feature [Microsoft-Hyper-V] with exit code: 14081
> 
> Automated error analysis suggests 14081 may mean The referenced assembly could not be found. Is a dependency missing?
> 
> (Automated response - build 895.)
> 
> **Edit: My VMs have an issue enabling Hyper-V that needs to be fixed.** 
> Edit2: I forgot that the [ WindowsFeatures](https://github.com/microsoft/winget-pkgs/labels/WindowsFeatures) Package needs a Windows Feature such as .NET Framework 3.5. label had this automation attached.

I'm not able to verify this installer and application.

**Pull request closed without merging**